.github/workflows/deploy.yml

```
@@ -68,8 +68,8 @@ jobs:
       - name: Get current app version
         run: echo "PRODUCTION_VERSION=$(npm run print-version --silent)" >> "$GITHUB_ENV"
 
-      - name: 🚀 Create release to trigger production deploy 🚀
-        run: gh release create ${{ env.PRODUCTION_VERSION }} --title ${{ env.PRODUCTION_VERSION }} --generate-notes
+      - name: 🚀 Edit the release to be no longer a prerelease to deploy production 🚀
+        run: gh release create ${{ env.PRODUCTION_VERSION }} --prerelease=false
```

Did you test this? Looking at the gh manual I would've expected something like this:

```bash
gh release edit ${{ env.PRODUCTION_VERSION }} --latest
```

---

Actually, coming from https://github.com/cli/cli/issues/6853#issuecomment-1384289577 it should be:

```suggestion
        run: gh release edit ${{ env.PRODUCTION_VERSION }} --prerelease=false
```

---

I did test this as it's written and it worked